### PR TITLE
Create a database from a seed file

### PR DESCRIPTION
**Pull request by glommer**
extend the creation command to add a --from-file flag that allows one to pass a sqlite file

### CODE REVIEW

File: internal/cmd/db_create.go [hunk 1]
```diff
@@ -70,6 +89,14 @@ var createCmd = &cobra.Command{
 			Password: res.Password,
 		}
 
+		if dbFile != nil {
```
Comment (jmwoliver) [inline internal/cmd/db_create.go:92]: I wonder if it makes a little more sense to have all the logic in one place after the database has been created?

```go
if dbFromFile != "" {
    dbFile, err := getDbFile(dbFromFile)
    if err != nil {
        return err
    }
    defer dbFile.Close()
    err := client.Databases.Seed(name, dbFile)
    if err != nil {
        client.Databases.Delete(name)
        return fmt.Errorf("could not create database %s: %w", name, err)
    }
}
```

Comment (jmwoliver) [reply internal/cmd/db_create.go:92]: Maybe it is better to check that the file exists first before creating the database though, I can see why you went that route instead.

Comment (glommer) [reply internal/cmd/db_create.go:92]: yeah, mostly we want to make sure that system-related errors don't even trigger a database creation event.

**Comment by athoscouto** [comment]
Closing in favor of https://github.com/chiselstrike/turso-cli/pull/361.
PRs from forks don't work well with our CI :(
I also made some changes! 👀

**Pull request closed without merging**